src/Installers/Windows/SharedFrameworkBundle/thm.wxl

```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <WixLocalization Culture="en-us" Language="1033" xmlns="http://schemas.microsoft.com/wix/2006/localization">
-  <String Id="Caption">[BundleNameFull] Setup</String>
-  <String Id="Title">[BundleNameShort]</String>
-  <String Id="SubTitle">[BundleNameSub]</String>
+  <String Id="Caption"><!--_locComment_text="{Locked='[BundleNameFull]'}"-->[BundleNameFull] Setup</String>
+  <String Id="Title"><!--_locComment_text="{Locked}"-->[BundleNameShort]</String>
+  <String Id="SubTitle"><!--_locComment_text="{Locked}"-->[BundleNameSub]</String>
```

Comment:
@cristianosuzuki77 is this the correct application of your suggestion in [ticket 710644, [SW] Need for loc comments in .wxl files?](https://ceapex.visualstudio.com/CEINTL/_workitems/edit/710644/)❔

Reply:
Yes, looks good.